euth/actions/models.py

```diff
@@ -43,6 +43,8 @@ class Action(models.Model):
     verb = models.CharField(max_length=255, db_index=True)
     description = models.TextField(blank=True, null=True)
 
+    recipients = models.TextField(blank=True, null=True)
```

Comment:
Why do we want to store the recipients of an action? Shouldn't the recipients be the same for each action?